frontend/src/views/Auth/Login/Login.scss

```diff
@@ -1 +1,40 @@
+.center {
+  padding: 0rem 1rem !important;
+}
 
+h2 {
+  width: 93px;
+  height: 38px;
+  font-family: OpenSans;
```

Comment:
About this properties like font we have a scss file, that has font properties, please import that file(Variables/Font.scss) and use it here.